### PR TITLE
Fix typo in spark-shell section in README

### PR DESCRIPTION
**Pull request by akirakw**
Fix README typo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,16 +27,16 @@ version: 1.2.0
 
 
 ## Using with Spark shell
-This package can be added to  Spark using the `--jars` command line option.  For example, to include it when starting the spark shell:
+This package can be added to  Spark using the `--packages` command line option.  For example, to include it when starting the spark shell:
 
 ### Spark compiled with Scala 2.11
 ```
-$SPARK_HOME/bin/spark-shell --packages com.databricks:spark-csv_2.11:1.2.0-s_2.11
```
Comment (falaki) [inline README.md]: This was not a typo. We need ```-s_2.11``` to specify scala version.

File: README.md [hunk 2]
```diff
@@ -27,16 +27,16 @@ version: 1.2.0
...
+$SPARK_HOME/bin/spark-shell --packages com.databricks:spark-csv_2.11:1.2.0
 ```
 
 ### Spark compiled with Scala 2.10
 ```
-$SPARK_HOME/bin/spark-shell --packages com.databricks:spark-csv_2.10:1.2.0-s_2.10
```
Comment (falaki) [inline README.md]: Same here.

File: README.md (hunk 1 above)
Comment (brkyvz) [inline README.md]: This is a bug with Spark Packages. the fix is correct.

File: README.md (hunk 2 above)
Comment (brkyvz) [inline README.md]: We're going to push the fix to Spark Packages today

Comment (falaki) [inline README.md]: Cool. I let's wait for it and re-run the tests.

Comment (brkyvz) [inline README.md]: Pushed the fix. Now the correct coordinate shows up:
`$SPARK_HOME/bin/spark-shell --packages com.databricks:spark-csv_2.11:1.2.0`

**Comment by JoshRosen** [comment]
I'm going to go ahead and merge this fix now, since the current (wrong) instructions are really confusing to people. Thanks for fixing this!

**Pull request closed without merging**